crates/language-tests/tests/language/statements/define/table/view/triggers/event_new_group.surql

```diff
@@ -18,7 +18,7 @@ value = "[{ after: { mean: 3f }, even: 'CREATE', id: test:triggered_on_view }]"
 {
 	DEFINE TABLE target;
 	DEFINE TABLE view AS SELECT math::mean(field) as mean FROM target GROUP ALL;
-
+	DEFINE TABLE test;
```

Comment:
I think this links back to an earlier comment, but why is a table definition needed here? DB is not strict